dotcom-rendering/src/components/FeatureCard.tsx

```diff
@@ -644,6 +688,24 @@ export const FeatureCard = ({
 											mainMedia={mainMedia}
 											isNewsletter={isNewsletter}
 										/>
+
+										{!isImmersive &&
+											mainMedia?.type === 'Audio' && (
+												<div
+													css={waveformStyles}
+													className="waveform"
+												>
+													<WaveForm
+														seed={
+															mainMedia.duration
+														}
+														height={64}
+														// Just enough to cover the full width of the feature card in it's largest form
+														bars={233}
```

Comment:
Could we do the isImmersive check in the bars param so we don't have to duplicate the full component? 

```suggestion
														bars={isImmersive ? 314 : 233}
```